### PR TITLE
Shooter tuning

### PR DESCRIPTION
**Pull request by hazeltorek**

### CODE REVIEW

**Review by JalenEvans — APPROVED** [review]
Very nice! I like the shooter code, it may just need a little more tuning in the future!
Thumbs Up

**Review by jboutin020106 — APPROVED** [review]
Good job 👍

File: src/main/java/frc/robot/subsystems/OuttakeSubsystem.java [hunk 1]
```diff
@@ -105,17 +112,54 @@ public double getSetpoint() {
...
+  }
+
+  private double getEncoderDelta() {
+    return encoder2.getIntegratedSensorPosition() + encoder1.getIntegratedSensorPosition();
+  }
+
+  public void resetEncoders() {
+    encoder1.setIntegratedSensorPosition(0, 10);
+    encoder2.setIntegratedSensorPosition(0, 10);
+  }
```
Comment (JalenEvans) [inline src/main/java/frc/robot/subsystems/OuttakeSubsystem.java]: No Java Docs

**Review by JalenEvans — CHANGES_REQUESTED** [review]

**Review by VZamora5 — APPROVED** [review]
very cool

**Review by JalenEvans — COMMENTED** [review]
Thank you for the java docs!
Thumbs Up

**Review by JalenEvans — APPROVED** [review]

**Pull request merged**